benchmarks/maxtext_v5p_model_configs.py

```diff
@@ -104,4 +104,93 @@
             + xla_flags_library.DATA_PARALLEL_OVERLAP
         ),
     ),
-)
\ No newline at end of file
+)
+
+llama4_scout_dropless_v5p_256  = _add_to_model_dictionary(
+    v5p_model_dict,
+    MaxTextModel(
+        model_name="llama4_scout_dropless_v5p_256",
+        model_type="llama4-17b-16e",
+        tuning_params={
+            "per_device_batch_size": 8,
+            "max_target_length": 8192,
+            "ici_fsdp_parallelism": -1,
+            "enable_checkpointing": False,
+            "dtype": "bfloat16",
+            "weight_dtype": "float32",
+            "megablox": True,
+            "sparse_matmul": True,
+            "dataset_type": "synthetic",
+            "opt_type": "adamw",
+            "skip_first_n_steps_for_profiler": 5,
+            "profiler_steps": 3,
+            "profiler": "xplane",
+            "remat_policy": "custom",
+            "decoder_layer_input": "offload",
+            "reuse_example_batch": 1,
+            "sa_block_q": 2048,
+            "sa_block_kv": 2048,
+            "sa_block_kv_compute": 2048,
+            "sa_block_q_dkv": 2048,
+            "sa_block_kv_dkv": 2048,
+            "sa_block_kv_dkv_compute": 2048,
+            "sa_block_q_dq": 2048,
+            "sa_block_kv_dq": 2048,
+            "tokenizer_path": "meta-llama/Llama-4-Maverick-17B-16E",
```

Comment:
Why does the Scout model use the Maverick tokenizer and the Maverick model use the Scout one?

Reply:
Good catch! It's a typo, and they are the same underneath: https://screenshot.googleplex.com/3ppVAgzFzZrFNAb